packages/indicators/jest.setup.js

```diff
@@ -10,3 +10,5 @@ afterAll(async () => {
   await clearTestData(database);
   await database.closeConnections();
 });
+
+jest.setTimeout(30000)
```

Comment:
Please ignore this in this commit, I've removed it from the later commit https://github.com/beyondessential/tupaia/pull/2125/commits/784f2f763be11ef180ba7de32039076912062a81